modules/add-book.js

```diff
@@ -0,0 +1,11 @@
+const BookName = document.querySelector('#book');
+const authorName = document.querySelector('#author');
+
+const bookList = JSON.parse(localStorage.getItem('book-list')) || [];
+export default function addBook() {
+  const title = BookName.value;
+  const author = authorName.value;
+  const books = { title, author };
+  bookList.push(books);
+  localStorage.setItem('book-list', JSON.stringify(bookList));
+}
```

Comment:
- [ ] Kindly ensure in all your files you use `arrow functions`: a new syntax for writing functions in JavaScript that provides a more concise way to define functions. This can make your code easier to read and write.

### Example of an Arrow function
`` const add = (a, b) => a + b; ``